src-ext/TypeScript/ggvega/src/encodingValue.ts

```diff
@@ -0,0 +1,100 @@
+import {contains} from './utils';
+/**
+ * Translate a ggplot2 size to a Vega-Lite size
+ *
+ * @remarks
+ * In the future, we may have to take the geom into account.
+ *
+ * **Called by**
+ * @see itmEncodingObjectByAesParamsObject
+ *
+ * @param size - `number`
+ *
+ * @returns `number`
+ */
+export function encodingValueSize(size: number): number {
```

Comment:
Could this take `string | number | undefined` then use `Number(size)`, etc., inside these functions?

Reply:
Yes. If the string isn't a number, It will return `NaN`.

Reply:
done